app/views.py

```diff
@@ -3,14 +3,58 @@
 '''
 
 
-from flask import Flask, request, render_template, redirect, url_for, flash
-from werkzeug import import_string, cached_property
+from flask import request, render_template, redirect, url_for, flash, send_from_directory
+from werkzeug.utils import secure_filename
 
 from app import app
+import os
 
 # NOTE: Flask defaults to serving html out of /templates, statics (js/css) out of /static
 
 @app.route('/', methods=['GET', 'POST'])
 def index():
     if request.method == 'GET':
         return render_template("index.html")
+
+def allowed_file(filename):
+    return '.' in filename and filename.rsplit('.', 1)[1].lower() == 'pdf'
+
+'''
+Enables viewing/editing of csv from pdf
+'''
+@app.route('/viewer/<filename>', methods=['GET'])
+def viewer(filename):
+    return render_template("viewer.html", filename=filename)
+
+'''
+Serves csv file to front-end
+'''
+@app.route('/uploads/<filename>')
```

Comment:
We're not persisting, so only one CSV file should be sent to the server at once, right? Do we want the data processing/rendering to happen server-side, or should the server just serve JS Code that handles visualization, etc? I think these are questions for @ma8642!

Reply:
That's actually a really good point @kevinjcliao Alex has not answered my email about whether the analysts will want to upload multiple pdfs yet, but I think for our preliminary product we should err on the side of simplicity -- one pdf at a time, no storing data.  We can also talk about this in our meeting on Thursday.